openapi.yaml

```diff
@@ -3027,6 +3140,40 @@ components:
           type: string
           description: "The phone of the contact"
           example: "402-555-1212"
+    ProjectCollaborator:
+      type: object
+      properties:
+        company_id:
+          type: string
+          description: "The unique ID of the collaborating company"
+          example: "48892885"
+        project_id:
+          type: string
+          description: "The unique ID of the project the collaborator is apart of"
+          example: "48892885"
+        permissions:
+          type: array
+          description: "The permissions for the collaborator"
+          items:
+            type: string
+        sharing_start_at:
+          type: integer
+          format: int32
+          description: "Timestamp when the project collaboration was started."
+        created_at:
+          type: integer
+          format: int32
+          description: "Timestamp when the project collaboration was created on the server"
+        updated_at:
+          type: integer
+          format: int32
+          description: "Timestamp when the project collaboration was last updated"
+    ProjectCollaborationInvite:
```

Comment:
`ProjectInvitation` would probably be a better type name here -- mirrors the database, the url path, and is unambiguously a noun and not a verb.